src/classes/STG_InstallScript_TEST.cls

```diff
@@ -244,4 +244,70 @@ public with sharing class STG_InstallScript_TEST {
             System.assertNotEquals(null, addr.Parent_Contact__c);
         }
     }
+
+    @isTest
+    public static void createAdminAccountForContactOnInstall() {
+
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe.getAdminAccRecTypeID()));
+
+        Contact cont = new Contact(
+                FirstName = 'Test',
+                LastName = 'User',
+                Email = 'usertest@domain.com'
+        );
+
+        ACCT_IndividualAccounts_TDTM.alreadyRunAfterInsert = true; // need to turn off the trigger for this test
```

Comment:
Forgot to mention that we are modifying the static flags in another pull request not merged yet. Using these static flags might require some extra efforts to deal with merge conflicts. Just quick fyi.